convert/src/label_studio_to_badgerdoc/badgerdoc_to_label_studio_use_case.py

```diff
@@ -24,12 +27,21 @@ class BadgerdocData(NamedTuple):
 
 
 class BDToLabelStudioConvertUseCase:
+    labelstudio_format = LabelStudioFormat()
+
     def __init__(
         self,
-        s3_client,
+        s3_client: BaseClient,
+        current_tenant: str,
+        token_data: TenantData,
     ) -> None:
         self.s3_client = s3_client
-        self.labelstudio_format = LabelStudioFormat()
+        self.current_tenant = current_tenant
+        self.token_data = token_data
+        self.request_headers = {
+            "X-Current-Tenant": self.current_tenant,
+            "Authorization": f"Bearer {self.token_data.token}",
```

Comment:
I believe **token data** should contain token type and should not be hardcoded as "Bearer". Please, doublecheck this.